.github/workflows/check.yml

```diff
@@ -77,9 +77,5 @@ jobs:
       - name: Install Python dependencies
         run: uv sync --all-extras --dev
         shell: bash
-      - name: Run ruff format
-        run: uv run ruff format --check .
-      - name: Run ruff lint
-        run: uv run ruff check --output-format=github .
```

Comment:
I'm not convinced it's a good idea to replace ruff with black and flake8 (which pycheck uses under the hood). Ruff is faster and more modern, and it's a single dependency to have; and I don't think it's a good idea to have different tooling in CI than in the local dev setup.

I'll approve because we want to get rid of the python part anyways, but I'm not convinced of this change.

Reply:
Hmm, it was not my intention to change any tooling but to use `just`, so that CI uses the same tooling as it is used locally.

Reply:
I'm only vaguely familiar with Python tooling and I'm using the setup that Johannes created. The only thing that I changed a while ago is to move from mypy to pyright. But I will check.